topics/materials-science/tutorials/muspinsim/workflows/7_Fitting.ga

```diff
@@ -0,0 +1,208 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Example of how we can fit some data with MuSpinSim",
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "Muon Spectroscopy Computational Project",
+            "url": "https://muon-spectroscopy-computational-project.github.io"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "name": "Muspinsim Example 9 - Fitting (imported from uploaded file)",
```

Comment:
This doesn't match the tutorial.
```suggestion
    "name": "Muspinsim Example 7 - Fitting (imported from uploaded file)",
```

This workflow also fails similarly to 6_Dissipation.ga